src/mbgl/style/source_impl.cpp

```diff
@@ -168,7 +172,8 @@ void Source::Impl::updateTiles(const UpdateParameters& parameters) {
                                    parameters.debugOptions & MapDebugOptions::Collision };
 
     for (auto& pair : tiles) {
-        pair.second->setPlacementConfig(config);
+        auto& tile = pair.second;
+        tile->setPlacementConfig(config);
```

Comment:
Intention is to improve readability (`tile` vs. `pair.second`).

If that's agreeable, we can move the object scope out of the for-loop so it only gets updated instead of created/deleted on each pass.